pkgs/applications/misc/furtherance/default.nix

```diff
@@ -0,0 +1,54 @@
+{ lib, stdenv, fetchFromGitHub, rustPlatform, appstream-glib, desktop-file-utils
+, glib, libadwaita, meson, ninja, pkg-config, python3, wrapGAppsHook4, dbus
+, gtk4, sqlite }:
+
+stdenv.mkDerivation rec {
+  pname = "furtherance";
+  version = "1.5.3";
+
+  src = fetchFromGitHub {
+    owner = "lakoliu";
+    repo = "Furtherance";
+    rev = "v${version}";
+    sha256 = "GKak1P5QX9Y7bXD3E+QPCbi1Auv1mgSYKgbLz7kc3NQ=";
+  };
+
+  patches = [
+    ./gnome-post-install.patch
```

Comment:
```suggestion
    # https://github.com/lakoliu/Furtherance/issues/79
    ./gnome-post-install.patch
```

Reply:
So in future builds, when this change is implemented on furtherance I just get rid of these lines, right?

Reply:
Exactly, yes. :-)